### PR TITLE
some comments in model

### PR DESCRIPTION
**Pull request by CurryNRice**

### CODE REVIEW

File: core/src/main/java/com/g16/feyrune/model/overworld/OverworldModel.java [hunk 1]
```diff
@@ -4,13 +4,17 @@
 import com.g16.feyrune.interfaces.IObserver;
 import com.g16.feyrune.model.overworld.encounter.Encounter;
 import com.g16.feyrune.model.overworld.encounter.EncounterHandler;
+import com.g16.feyrune.model.overworld.map.Map;
```
Comment (x183) [inline core/src/main/java/com/g16/feyrune/model/overworld/OverworldModel.java:7]: Why is this import added?

**Review by x183 — APPROVED** [review]
Looks good

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]

**Review by Supergamer1337 — APPROVED** [review]
I fixed the thing, someone else look

File: core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java [hunk 2]
```diff
@@ -5,6 +5,9 @@
 
 import java.awt.*;
 
+/**
+ * This class is responsible for transelateing an inpout direction from the controller to a new position for the player
```
Comment (Krysset) [inline core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java:9]: typo

File: core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java [hunk 3]
```diff
@@ -75,6 +81,10 @@ private Point adjustDirectionForCollision(Point coordinates, Map map) {
         return dir;
     }
 
+
+    /**
+     * Chekes if player is allowed to move accoring to time passed
```
Comment (Krysset) [inline core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java:86]: typo

Comment (Krysset) [reply core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java:86]: 2x

File: core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java (hunk 2 above)
Comment (Krysset) [reply core/src/main/java/com/g16/feyrune/model/overworld/MovementHandler.java:9]: 2x

File: core/src/main/java/com/g16/feyrune/model/StateHandler.java [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 import com.g16.feyrune.enums.ModelState;
 
+/**
+ * This class handles the model state
+ * the model state is what decides witch controller and view that should be active,
+ * and witch part of the model that should get polled
```
Comment (Krysset) [inline core/src/main/java/com/g16/feyrune/model/StateHandler.java:8]: typo 2x on witch

**Review by x183 — APPROVED** [review]
100% more sexy now

**Review by Krysset — APPROVED** [review]
Looks good very happy 😎

**Pull request merged**